environs/bootstrap/bootstrap.go

```diff
@@ -655,19 +655,52 @@ func isCompatibleVersion(v1, v2 version.Number) bool {
 	return v1.Compare(v2) == 0
 }
 
-// setPrivateMetadataSources sets the default tools metadata source
-// for tools syncing, and adds an image metadata source after verifying
-// the contents.
+// setPrivateMetadataSources verifys the metadataDir and uses it to
+// set the default tools metadata source for tools syncing, and adds
```

Comment:
s/tools/agent binaries/

I realize we're currently looking for a directory ending in 'tools' but telling people 'we find agent binaries in a directory named tools' is still clearer what we are actually looking for. 'tools' could be anything (such as helper commands, eg juju-metadata, etc).